### PR TITLE
Add IDTk in the corresponding pages

### PR DESCRIPTION
**Pull request by lauportell**
Add IDTk link in:
- Human pathogen data
- COVID-19 Data Portal

### CODE REVIEW

File: pages/tool_assembly/covid19_data_portal.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ description: The COVID-19 Data Portal brings together relevant datasets for shar
...
... Portal. The COVID-19 Data Portal brings together and continuously updates relevant COVID-19 datasets from a breadth of analytical platforms. Data are submitted using the SARS-CoV-2 Data Hubs functions or via other major centres of biomedical data. The data available from the COVID-19 Data Portal cover raw and assembled viral and human sequences, protein structures, proteomics, gene and protein expression data, compound screening, metabolomics and imaging data (from {% tool "elixir-core-data-resources" %} and other databases); COVID-19-relevant literature publications and pre-prints are also integrated. The aim is to have a wide variety of open data from across the globe systematised and easily accessible to researchers following FAIR principles. The European COVID-19 Data Platform enables national data producers to share biomolecular data with the international scientific community, making these data available for reuse. Ultimately it aims to allow for rapid analysis and dissemination to inform research, public health and health communities and in an evidenced-based manner. For detailed information on managing infectious disease data, you can always refer to the {% tool "idtk" %}.
```
Comment (bianchini88) [inline pages/tool_assembly/covid19_data_portal.md:15]:
```suggestion
The European COVID-19 Data Portal is part of the European COVID-19 Data Platform, which was launched to facilitate the urgent need to share and analyse COVID-19 data and thus accelerate research that will provide responses and build solutions, such as vaccines, treatments and public health interventions. The Platform comprises three core components: the SARS-CoV-2 Data Hubs, the Federated European Genome-phenome Archive and the COVID-19 Data Portal. The COVID-19 Data Portal brings together and continuously updates relevant COVID-19 datasets from a breadth of analytical platforms. Data are submitted using the SARS-CoV-2 Data Hubs functions or via other major centres of biomedical data. The data available from the COVID-19 Data Portal cover raw and assembled viral and human sequences, protein structures, proteomics, gene and protein expression data, compound screening, metabolomics and imaging data (from {% tool "elixir-core-data-resources" %} and other databases); COVID-19-relevant literature publications and pre-prints are also integrated. The aim is to have a wide variety of open data from across the globe systematised and easily accessible to researchers following the FAIR principles. The European COVID-19 Data Platform enables national data producers to share biomolecular data with the international scientific community, making these data available for reuse. Ultimately, it aims to allow for rapid analysis and dissemination to inform research, public health and health communities in an evidence-based manner. For detailed information on managing infectious disease data, you can always refer to the {% tool "idtk" %}.
```

**Review by bianchini88 — APPROVED** [review]
I suggested a couple of style/grammar improvements.

**Comment by Nazeeefa** [comment]
Thanks @lauportell and @bianchini88 - are all comments resolved/addressed? :)

File: pages/tool_assembly/covid19_data_portal.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ description: The COVID-19 Data Portal brings together relevant datasets for shar
...
... Portal. The COVID-19 Data Portal brings together and continuously updates relevant COVID-19 datasets from a breadth of analytical platforms. Data are submitted using the SARS-CoV-2 Data Hubs functions or via other major centres of biomedical data. The data available from the COVID-19 Data Portal cover raw and assembled viral and human sequences, protein structures, proteomics, gene and protein expression data, compound screening, metabolomics and imaging data (from {% tool "elixir-core-data-resources" %} and other databases); COVID-19-relevant literature publications and pre-prints are also integrated. The aim is to have a wide variety of open data from across the globe systematised and easily accessible to researchers following the FAIR principles. The European COVID-19 Data Platform enables national data producers to share biomolecular data with the international scientific community, making these data available for reuse. Ultimately, it aims to allow for rapid analysis and dissemination to inform research, public health and health communities in an evidence-based manner. For detailed information on managing infectious disease data, you can always refer to the {% tool "idtk" %}.
```
Comment (diana-pilvar) [inline pages/tool_assembly/covid19_data_portal.md:15]: Split the first sentence to make easier read
The European COVID-19 Data Portal is part of the European COVID-19 Data Platform, which was launched to facilitate the urgent need to share and analyse COVID-19 data.
The portal will accelerate research that will provide responses and build solutions, such as vaccines, treatments and public health interventions.

**Comment by bianchini88** [comment]
@Nazeeefa All good from me.

File: pages/tool_assembly/covid19_data_portal.md (hunk 2 above)
Comment (Nazeeefa) [reply pages/tool_assembly/covid19_data_portal.md:15]: @lauportell can you kindly make this suggested change? thanks

Comment (lauportell) [reply pages/tool_assembly/covid19_data_portal.md]: done!

**Pull request merged**